internal/proto5server/server_getmetadata_test.go

```diff
@@ -357,6 +357,10 @@ func TestServerGetMetadata(t *testing.T) {
 				return got.EphemeralResources[i].TypeName < got.EphemeralResources[j].TypeName
 			})
 
+			sort.Slice(got.ListResources, func(i int, j int) bool {
+				return got.ListResources[i].TypeName < got.ListResources[j].TypeName
+			})
```

Comment:
You might have noticed random CI failures for actions because I forgot to add this 😆 

It'll be added in #1186, but in the mean time you can just re-run the CI and it'll eventually pass 😆